frontend/app/utils/date-utils.ts

```diff
@@ -37,6 +37,41 @@ export function parseDateString(dateString: string, validate: boolean = true): {
   };
 }
 
+/**
+ * Checks if a string is a validad date. Else returns false.
+ * @param dateString The date string to parse.
+ * @param pattern Format to check against.
+ * @param delimitter What to splice the dateString by.
+ * @returns Returns a boolean.
+ */
+export function isValidDate(dateString: string, pattern: string, delimitter: string): boolean {
```

Comment:
This can already be acheive with date-fns.

```js
const dateString = '1985-01-10';
const parsedDate = parse('dateString', 'yyyy-MM-dd', new Date());
const result = isValid(new Date(2014, 1, 31));
//=> true
```

https://date-fns.org/v3.6.0/docs/parse
https://date-fns.org/v3.6.0/docs/isValid

Reply:
- [x] Agree

Reply:
It's not needed and I recommend we remove it and stick to the established patterns. This will cause confusions.